engine/src/main/java/org/terasology/rendering/world/WorldRenderer.java

```diff
@@ -33,7 +32,7 @@
     void onChunkUnloaded(Vector3i chunkPos);
 
     public enum WorldRenderingStage {
-        DEFAULT,
+        MONO,
         OCULUS_LEFT_EYE,
         OCULUS_RIGHT_EYE
```

Comment:
Renaming the enum constants to the more general version sounds reasonable to me. Eyes are not necessarily linked to Oculus :smile: It also shouldn't break existing code.